content/blog/workflows-0-2-0/index.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+output: hugodown::hugo_document
+slug: workflows-0-2-0
+title: workflows 0.2.0
+date: 2020-09-16
+author: Davis Vaughan
+description: >
+    workflows 0.2.0 is now on CRAN!
+photo:
+  url: https://unsplash.com/photos/nN5L5GXKFz8
+  author: Mark Fletcher-Brown
+categories: [package] 
+tags: [tidymodels]
+editor_options: 
+  chunk_output_type: console
+---
+
+We're excited to announce the release of [workflows](https://workflows.tidymodels.org/) 0.2.0. workflows is a [tidymodels](https://www.tidymodels.org/) package for bundling a model specification from [parsnip](https://parsnip.tidymodels.org/) with a preprocessor, such as a formula or [recipe](https://recipes.tidymodels.org/). Doing this can streamline the model fitting workflow and combines nicely with [tune](https://tune.tidymodels.org/) for performing hyperparameter tuning.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("workflows")
+```
+
+```{r setup, warning=FALSE}
+library(workflows)
+library(parsnip)
+```
+
+## Variables
+
+The main change in this release of workflows is the introduction of a new preprocessor method: `add_variables()`. This adds a third method to specify model terms, in addition to `add_formula()` and `add_recipe()`. 
+
+`add_variables()` has a tidyselect interface, where `outcomes` are specified using bare column names, followed by `predictors`.
+
+```{r}
+linear_spec <- linear_reg() %>%
+  set_engine("lm")
+
+wf <- workflow() %>%
+  add_model(linear_spec) %>%
+  add_variables(mpg, c(cyl, disp))
```

Comment:
`add_variables(mpg, predictors = c(cyl, disp))` or add both argument names if that looks weird.

Reply:
I tried to preface with

```
`outcomes` are specified using bare column names, followed by `predictors`
```

because i figured you'd say something like this 😆